app.js

```diff
@@ -103,7 +113,7 @@ io.on('connection', function(socket){
 
     var term;
     if (process.getuid() == 0) {
```

Comment:
As it stands, this change would only work when running wetty as root. Adding the change would make it work regardless of the user.